src/MAUI/Maui.Samples/Samples/Layers/CustomFeatureClustering/CustomFeatureClustering.xaml

```diff
@@ -0,0 +1,87 @@
+﻿<ContentPage x:Class="ArcGIS.Samples.CustomFeatureClustering.CustomFeatureClustering"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esriTK="clr-namespace:Esri.ArcGISRuntime.Toolkit.Maui;assembly=Esri.ArcGISRuntime.Toolkit.Maui"
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
+    <Grid ColumnDefinitions="*, Auto" RowDefinitions="Auto, *">
+        <esriUI:MapView x:Name="MyMapView" Style="{DynamicResource EsriSampleGeoView}" />
+        <Border Style="{DynamicResource EsriSampleControlPanel}">
```

Comment:
Control panel is rendering 90% off screen for me on iPhone XS.

Reply:
I'll run this on iOS shortly to see if I can find a solution.

Reply:
I should've added - this is an issue when rotating device orientation to landscape.

Reply:
I've tested it. As it works and displays ok in portrait I think it's ok for now as I don't think the fix is a small piece of effort.